docs/Readme.md

```diff
@@ -32,7 +32,7 @@ This template set deploys the following infrastructure core to your LAMP instanc
 - Controller VM running cron and handling syslog for the autoscaling cluster
 - [Azure Load balancer](https://azure.microsoft.com/en-us/services/load-balancer/) to balance across the autoscaling instances
 - [Azure Database for MySQL](https://azure.microsoft.com/en-us/services/mysql/) or [Azure Database for PostgreSQL](https://azure.microsoft.com/en-us/services/postgresql/) or [Azure SQL Database](https://azure.microsoft.com/en-us/services/sql-database/)
-- Dual [GlusterFS](https://www.gluster.org/) nodes or NFS for highly available access to LAMP files
+- [AzureFiles](https://docs.microsoft.com/en-us/azure/storage/files/) or NFS for highly available access to LAMP files
```

Comment:
Let's call out that we use Azure Premium Files instead of "Azure Files". AzureFiles doesn't really mean much.